boudicca.base/common-model/src/main/kotlin/base/boudicca/model/structured/AbstractKey.kt

```diff
@@ -26,7 +26,7 @@ abstract class AbstractKey<T : AbstractKey<T>>(val name: String, variants: List<
 
     companion object {
         val COMPARATOR = compareBy<AbstractKey<*>> { it.name }.thenComparing { o1, o2 ->
-            (0..<min(o1.variants.size, o2.variants.size)).forEach { i ->
+            for (i in 0..<min(o1.variants.size, o2.variants.size)) {
```

Comment:
why?

Reply:
because there is a rule in detekt which says you should not do that and i agree :D